docs/getting_started.md

```diff
@@ -0,0 +1,24 @@
+## Requirements:
+* Django 2.2
+* Wagtail 2.2
+* Python 3
```

Comment:
Python 3 is required by Wagtail 2+ anyway

Reply:
Good point, I've removed now.